node/module_all.ts

```diff
@@ -116,6 +116,8 @@ export default {
   "internal/util": internalUtil,
   net,
   os,
+  "path/posix": path,
```

Comment:
I think we need to pass something like the below here:

```ts
import pathPosix from "./path/posix.ts";
```
And that means we need to create default export of `path/posix.ts`

```ts
export default {
  sep,
  delimiter,
  ...
}
```
We probably also need to update `path/mod.ts` like the below

```diff
-import * as _posix from "./posix.ts";
+import _posix from "./posix.ts";
```